components/tekton-ci/base/repository.yaml

```diff
@@ -103,3 +103,10 @@ metadata:
   name: quality-dashboard
 spec:
   url: "https://github.com/redhat-appstudio/quality-dashboard"
+---
```

Comment:
Will it interfere with the existing [tekton pipelines](https://github.com/redhat-appstudio/ci-helper-app/tree/main/.tekton) in the repo?

Reply:
good question, these pipelines manifests will be removed in a follow-up PR